src/main/java/org/example/LessonDao.java

```
@@ -91,20 +94,26 @@ public Lesson getLessonById(int lessonId) throws SQLException {
 
             try (ResultSet resultSet = statement.executeQuery()) {
                 if (resultSet.next()) {
-                    int lessonIdResult = resultSet.getInt(1);
-                    String lessonName = resultSet.getString(2);
-                    int homeworkId = resultSet.getInt(3);
-                    String homeworkName = resultSet.getString(4);
-                    String homeworkDescription = resultSet.getString(5);
+                    int lessonIdIndex = 1;
+                    int lessonNameIndex = 2;
+                    int homeworkIdIndex = 3;
+                    int homeworkNameIndex = 4;
+                    int homeworkDescriptionIndex = 5;
+
+                    int lessonIdResult = resultSet.getInt(lessonIdIndex);
+                    String lessonName = resultSet.getString(lessonNameIndex);
+                    int homeworkId = resultSet.getInt(homeworkIdIndex);
+                    String homeworkName = resultSet.getString(homeworkNameIndex);
+                    String homeworkDescription = resultSet.getString(homeworkDescriptionIndex);
 
                     Homework homework = new Homework(homeworkId, homeworkName, homeworkDescription);
-                    return new Lesson(lessonIdResult, lessonName, homework);
+                    lesson = new Lesson(lessonIdResult, lessonName, homework);
                 }
             }
         } catch (SQLException e) {
-            throw new SQLException("Error retrieving lesson: " + e.getMessage());
+            System.err.println("Error retrieving lesson: " + e.getMessage());
```

throw an exception instead of using a print statement